search_service/models/table.py

```diff
@@ -8,38 +8,46 @@
 
 from .base import Base
 from search_service.models.tag import Tag
+import time
 
 
 @attr.s(auto_attribs=True, kw_only=True)
 class Table(Base):
     """
     This represents the part of a table stored in the search proxy
     """
-    database: str
-    cluster: str
-    schema: str
-    name: str
-    key: str
+    id: str
+    database: Optional[str] = None
+    cluster: Optional[str] = None
+    schema: Optional[str] = None
+    name: Optional[str] = None
+    key: Optional[str] = None
     display_name: Optional[str] = None
-    tags: List[Tag]
-    badges: List[Tag]
+    tags: List[Tag] = []
+    badges: List[Tag] = []
```

Comment:
typically we put None for list default value (e.g https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments)

Reply:
sounds good, I just wanted to be consistent with original code